python_modules/dagster/dagster/_core/definitions/dependency.py

```diff
@@ -673,33 +673,18 @@ def the_job():
 
 
     Args:
-        solid (str): (legacy) The name of the solid that is depended on, that is, from which the value
+        node (str): The name of the node (op or graph) that is depended on, that is, from which the value
             passed between the two nodes originates.
         output (Optional[str]): The name of the output that is depended on. (default: "result")
         description (Optional[str]): Human-readable description of this dependency.
-        node (str): The name of the node (op or graph) that is depended on, that is, from which the value
-            passed between the two nodes originates.
     """
 
     def __new__(
         cls,
-        solid: Optional[str] = None,
+        node: str,
```

Comment:
this is a top level exported API so I think this breaking change needs to wait for `1.2`

Reply:
I believe we'd previously decided with 1.0 that lingering `solid` or `pipeline` args on public symbols could be deleted at any time, but perhaps I'm misremembering-- @sryza what do you think

Reply:
I agree with @smackesey on this. @alangenfeld - feel free to push back though.

Reply:
My main concern here is that the docs example that I expect people may have copy pasted to start from was expclitly using `solid=` up until this PR so I think has a higher chance of breaking people.

Reply:
that sounds reasonable to me

Reply:
>I think we should update the docs and swap node in to first position in this PR and then stack a 1.2 PR to remove solid from the constructor.

done